<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inserisci_Data_di_Oggi_Window</class>
 <widget class="QWidget" name="Inserisci_Data_di_Oggi_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Scadenza</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-10</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 140, 255);</string>
   </property>
   <property name="text">
    <string>Inserisci Data di Oggi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);
                 color: rgb(255, 255, 255);
                 border-radius:10px;
             </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img_resources.qrc">
     <normaloff>:/cancel/res/cancel.png</normaloff>:/cancel/res/cancel.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="avanti_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);

color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Avanti</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="data_oggi_dateTimeEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateTimeEdit::down-button {


     image: url(:/arrows/res/down_arrow.png);
     width:50px;


     }


     QDateTimeEdit::up-button {


     image: url(:/arrows/res/up_arrow.png);
     width: 50px;
     }


     QDateTimeEdit {
     color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
     border-radius:25px;
border: 5px solid rgb(0,140,255);


}</string>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>10</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QPushButton" name="keyboard_btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);

color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img_resources.qrc">
     <normaloff>:/keyboard/res/keyboard_icon.png</normaloff>:/keyboard/res/keyboard_icon.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancel_btn_</sender>
   <signal>clicked()</signal>
   <receiver>Inserisci_Data_di_Oggi_Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
